docs/variants/pc_engines/firmware-update.md

```diff
@@ -0,0 +1,51 @@
+# Legacy -> UEFI Update
+
+This document describes the process of updating the platform's legacy SeaBIOS
+to Dasharo firmware based on EDK II.
+
+## OS compatibility
+
+Switching from BIOS to UEFI-based firmware might cause compatibility issues if
+you already had an operating system installed on your platform. Some OSs can
+handle the switch without any issues, while others may need to be reinstalled.
+We have tested the update on several operating systems. The results are
+presented in the table below:
+
+### Ubuntu
```

Comment:
Can it be summarized in a table with green tick / red cross (or similar) icons depending on the support?